src/components/Book.css

```diff
@@ -0,0 +1,100 @@
+.bookContainer {
+  display: flex;
+  border-radius: 10px;
+  background: #f4ecec;
+  box-shadow: 6px 6px 12px #beb8b8, -6px -6px 12px #fff;
+  padding: 20px 30px;
+  justify-content: space-between;
+}
+
+.bookDetails {
+  display: flex;
+  flex-direction: column;
+}
+
+.bookCategory {
+  color: rgb(87, 87, 87);
+}
+
+.bookTitle {
+  color: #063b69;
+}
+
+.bookAuthor {
+  color: #4386bf;
+  font-weight: bold;
+}
+
+.bookRemoveButton {
+  border: none;
+  font-size: medium;
+  background: none;
+  color: #4386bf;
+  cursor: pointer;
+}
+
+.bookCommentsLink {
+  cursor: pointer;
+  color: #4386bf;
+}
+
+.bookEditLink {
+  cursor: pointer;
+  color: #4386bf;
+}
+
+.completedness {
+  display: flex;
+  gap: 15px;
+}
+
+.circular-progress {
+  position: relative;
+  height: 100px;
+  width: 100px;
+  background: conic-gradient(#4395fc 0deg, #ededed 0deg);
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.circular-progress::before {
+  content: '';
+  position: absolute;
+  height: 80px;
+  width: 80px;
+  background-color: white;
+  border-radius: 50%;
+}
+
+.progress-value {
+  position: relative;
+  font-size: 1.5em;
+  font-weight: 600;
+  color: #4395fc;
+}
+
+.percentage {
+  display: flex;
+  flex-direction: column;
+  gap: 6px;
+  justify-content: center;
```

Comment:
- [ ] For DRY CSS kindly combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Please check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)